### DISCUSSION TITLE
How to install it ?

### DISCUSSION THREAD

**Discussion by RYG81** [AI4Bharat/Chitralekha#9] [Q&A]
Can someone please guide on how to install it?
It looks quite good project, but I am struggling to install it.

Please help
Rahul

**Comment by aravinth** [comment]
Hi @RYG81 - The parent repo contains links to backend & frontend independently. Under each of these repos you can find the setup instructions in the README.

**Reply by vvonchain** [reply]
there are too many setups in backend; even after following the readme of backend; it doesn't work as expected 

Can u help what is important in env config to run it smoothly?

**Comment by arihant2301** [comment]
Have tries the instructions. Not very easy to follow for a newbie. I could not complete the setup with those instructions.

**Comment by RYG81** [comment]
need better doc on installing it. Please guide.
